glib/src/subclass/object.rs

```diff
@@ -471,6 +471,13 @@ mod test {
         assert_eq!(properties[3].name(), "child");
     }
 
+    #[test]
+    fn test_auto_send_sync() {
+        fn check<T: Send + Sync>() {}
+
+        check::<ChildObject>();
```

Comment:
@GuillaumeGomez Once your trycompile test stuff is in you could also add a test that `check::<SimpleObject>()` fails to compile

Reply:
You can already use it. Or do you mean something else?

Reply:
We don't depend on it yet

Reply:
Since it's a `dev-dependences`, you have my blessing. :)

Reply:
Done